examples/usecases/ranking_with_multitask_learning.ipynb

```diff
@@ -50,7 +50,7 @@
     "\n",
```

Comment:
<p>can we write instead "In this example notebook, we use a synthetic dataset generated from the schema of the dataset released in the <a href="https://arxiv.org/abs/2210.10629" target="_blank">TenRec paper</a>, ..." ?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/models/pull/998/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>